CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Change history for ui-oai-pmh
+
+## 1.0.0 (IN PROGRESS)
+
+* New app created with stripes-cli, OAI-PMH blank UI module is created. Refs MODOAIPMH-94.
```

Comment:
I saw a tendency to use the present time in the changelog. I am following that way in data-export modules. Consider doing it as well.

Reply:
Fixed